### ISSUE TITLE
Include problems with komi

### ISSUE THREAD

**Issue by PJTraill** [antonTobi/counting#2]
Some contributors to the OGS forum thread at https://forums.online-go.com/t/a-little-game-i-made-to-practice-counting-quickly/30103/11 felt that some problems should include komi.

This would need to be shown with each problem and, if issue #1 is implemented, when the correct scores was shown.

**Comment by antonTobi** [comment]
I chose not to include komi for a couple reasons:

- I wanted to focus fully on developing the skill of counting points on the board - adding on komi doesn't really present any interesting extra challenge for the players, it's just an additional number to keep track of.
- As it works now, the problems can be color-reversed without issue. This means that I could create all the problems with a score of B+1 and then choose randomly whether to color-reverse each board when playing. If komi were included, this would require a whole new set of positions to be created, half of them white wins and half of them black wins.

So personally I'm not motivated to add in komi, but of course anyone is welcome to create a fork for that purpose!

**Comment by PJTraill** [comment]
I agree it would not add much; I just thought it worth  recording the various suggestions.
